compact/ecal/bic/bic.xml

```diff
@@ -39,6 +39,15 @@
     <constant name="EcalBarrel_Calorimeter_offset"
       value="(EcalBarrel_Calorimeter_zmax - EcalBarrel_Calorimeter_zmin)/2.0"/>
 
+    <constant name="EcalBarrel_LightGuide_length"      value="5*cm"/>
+    <constant name="EcalBarrel_EndOfSectorBox_length"  value="12*cm"/>
+    <constant name="EcalBarrel_LightGuide_zmin"        value="EcalBarrel_Calorimeter_zmin + EcalBarrel_LightGuide_length"/>
+    <constant name="EcalBarrel_LightGuide_zmax"        value="EcalBarrel_Calorimeter_zmax + EcalBarrel_LightGuide_length"/>
+    <constant name="EcalBarrel_EndOfSectorBox_zmin"    value="EcalBarrel_Calorimeter_zmin + EcalBarrel_EndOfSectorBox_length"/>
+    <constant name="EcalBarrel_EndOfSectorBox_zmax"    value="EcalBarrel_Calorimeter_zmax + EcalBarrel_EndOfSectorBox_length"/>
+    <constant name="EcalBarrel_LightGuide_eGoing_posZ" value="-EcalBarrel_LightGuide_zmin"/>
+    <constant name="EcalBarrel_LightGuide_pGoing_posZ" value="EcalBarrel_LightGuide_zmax"/>
```

Comment:
There is no precedent for `eGoing` or `pGoing`. We use by convention only "positive" and "negative" end caps, defined by the sign of z.

FYI @veprbl for input.

Reply:
I think, we usually do electron and ion or negative and positive, like you point out.

Reply:
Hi Minho, just implement one, and ask for review. This is not a major change.